config/default/eu_cookie_compliance.settings.yml

```diff
@@ -54,9 +54,9 @@ cookie_name: ''
 exclude_uid_1: false
 better_support_for_screen_readers: false
 method: categories
-disabled_javascripts: 'google_analytics:modules/contrib/localgov_eu_cookie_compliance/js/deferred-ga-script-runner.js'
+disabled_javascripts: "analytics_cookies:modules/contrib/localgov_eu_cookie_compliance/js/deferred-ga-script-runner.js\r\nanalytics_cookies:modules/contrib/localgov_eu_cookie_compliance/js/deferred-hotjar-script-runner.js"
```

Comment:
@siliconmeadow We're not using the hotjar script, we're bringing it in with GTM. Same with the ga runner, actually.